js/src/map/Map.js

```diff
@@ -0,0 +1,286 @@
+var widgets = require('@jupyter-widgets/base');
```

Comment:
I would not put `Map.js` alone in a directory, I guess it's fine to put it inside of `js/src/` directory